less/variables.less

```diff
@@ -11,4 +11,6 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+@phone: ~"(max-width: 500px)";
```

Comment:
Task 5: Mobile Updates Needed - WELL DONE!
 Use escaping to create a variable named @mobile that contains this value: (max-width: 500px). Use the variables.less file to house your styling. - DONE!
 Review the mobile design file. You will see several design updates that need updating. - DONE!
 Match the design file at 500px as well as you can